code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -1536,3 +1536,52 @@
 	icon = 'icons/clothing/uniform/costume/spiral.dmi'
 	icon_state = "kaminacape"
 	worn_render_flags = WORN_RENDER_SLOT_ONE_FOR_ALL
+
+//The Chippin' In Set -Cap
+/obj/item/clothing/suit/storage/toggle/heated
+	name = "Runner Jacket"
+	desc = "A sturdy high-vis jacket patterned after a lost society's first responders. It has been marked with unfamiliar graffiti on the back."
+	icon_state = "runner_jacket"
+	inv_hide_flags = HIDEHOLSTER
+	body_cover_flags = UPPER_TORSO|LOWER_TORSO|ARMS
+	cold_protection = UPPER_TORSO|LOWER_TORSO|ARMS|LEGS
+	min_cold_protection_temperature = SPACE_SUIT_MIN_COLD_PROTECTION_TEMPERATURE
```

Comment:
why would a Jacket keep you warm in 2 kelvin weather?

Reply:
```suggestion
	min_cold_protection_temperature = ARMOR_MIN_COLD_PROTECTION_TEMPERATURE
```
160 is still hilarious but more reasonable on a logic level
(you can also toggle the min_cold_protection_temperature with the opening and closing of the buttons/zippers

Reply:
I based these values off of the baseline Winter Coat, in fact. The winter coat that every department gets in multiple variants, that is available on the map in many locations. If we nerf this coat, we'll need to nerf literally all winter coats as well. They run on the same principles.